backend/src/test/java/com/woowacourse/gongcheck/core/application/JobServiceTest.java

```diff
@@ -65,272 +70,518 @@ class JobServiceTest {
+                List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
+                request = new JobCreateRequest("청소", sections);
+            }
+
+            @Test
+            void 예외가_발생한다() {
+                assertThatThrownBy(() -> jobService.createJob(host.getId(), 0L, request))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 공간입니다.");
+            }
+        }
     }
 
-    @Test
-    void Space가_존재하지_않는데_Job_생성_시_예외가_발생한다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
-
-        List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
-        List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
-        JobCreateRequest jobCreateRequest = new JobCreateRequest("청소", sections);
-
-        assertThatThrownBy(() -> jobService.createJob(host.getId(), 0L, jobCreateRequest))
-                .isInstanceOf(NotFoundException.class)
-                .hasMessage("존재하지 않는 공간입니다.");
-    }
 
     @Nested
-    class Job을_수정한다 {
-        Host host;
-        Space space;
-        List<TaskCreateRequest> tasks;
-        List<SectionCreateRequest> sections;
-        JobCreateRequest jobCreateRequest;
-        Long savedJobId;
-
-        @BeforeEach
-        void setUp() {
-            host = hostRepository.save(Host_생성("1234", 1234L));
-            space = spaceRepository.save(Space_생성(host, "잠실"));
-            tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
-            sections = List.of(new SectionCreateRequest("대강의실", tasks));
-            jobCreateRequest = new JobCreateRequest("청소", sections);
-            savedJobId = jobService.createJob(host.getId(), space.getId(), jobCreateRequest);
+    class updateJob_메소드는 {
+
+        @Nested
+        class 기존의_Job이_존재하는_경우 {
+
+            private Host host;
+            private JobCreateRequest request;
+            private Long savedJobId;
+
+            @BeforeEach
+            void setUp() {
+                host = hostRepository.save(Host_생성("1234", 1234L));
+                Space space = spaceRepository.save(Space_생성(host, "잠실"));
+                List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
+                List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
+                request = new JobCreateRequest("청소", sections);
+                savedJobId = jobService.createJob(host.getId(), space.getId(), request);
+            }
+
+            @Test
+            void 기존의_존재하는_Job을_삭제한_후_새로운_Job을_생성하여_수정한다() {
+                Long updateJobId = jobService.updateJob(host.getId(), savedJobId, request);
+
+                assertThat(updateJobId).isNotNull();
+            }
```

Comment:
```suggestion
            @Test
            void 기존에_존재하던_Job을_삭제한_후_새로운_Job을_생성한다() {
                Long updateJobId = jobService.updateJob(host.getId(), savedJobId, request);

                assertThat(updateJobId).isNotNull();
            }
```

이렇게 해도 괜찮지 않을까요? 엄밀히 말하면 수정하는게 아니니까요!

테스트 이름에 기존에 존재하는 Job을 삭제한 후, 라고 되어있는데 이 부분에 대한 검증도 필요하지 않을까요?